### PR TITLE
fix: same management token in live preview for different stack objects fixed

### PR DESCRIPTION
**Pull request by nadeem-cs**

### CODE REVIEW

File: src/core/modules/entry.js [hunk 1]
```diff
@@ -284,6 +284,14 @@ export default class Entry {
...
 
+    /**
+     * @method includeOwner
+     * @memberOf Entry
+     * @description Include the owner details along with the entry/entries details.
+     * @example stack.ContentType(contentType_uid).Entry(entry_uid).includeOwner().fetch()
+     * @returns {Entry}
+     * @instance
+     */
     includeOwner() {
```
Comment (ishaileshmishra) [inline src/core/modules/entry.js:295]: Check changelog it might be deprecated in past. @nadeem-cs

Comment (nadeem-cs) [reply src/core/modules/entry.js:296]: Thanks for the correction. Marked as deprecated.

**Review by ishaileshmishra — CHANGES_REQUESTED** [review]
Added few suggestion @nadeem-cs

File: index.d.ts [hunk 2]
```diff
@@ -55,6 +55,7 @@ export interface Config {
     live_preview?: LivePreview;
     plugins?: ContentstackPlugin[];
     fetchOptions?: FetchOptions;
+    ea_headers?: string[]
```
Comment (ishaileshmishra) [inline index.d.ts:58]: Is it public or private, Is user able to see this variable ? if that is the case then make this variable early_access?

Comment (ishaileshmishra) [reply index.d.ts:58]: - Could you please rename the variable "header-ea" to "early_access"? It's important to use a more explicit name as users may not readily understand the abbreviation for "early access." We also don't need to disclose the internal handling of user input like how we are handling it.

- I'm not able to locate the tests based on early access implementation.

Rest look good to me @nadeem-cs

Comment (nadeem-cs) [reply index.d.ts]: Done with the changes. @ishaileshmishra

**Review by ishaileshmishra — APPROVED** [review]
approved

**Pull request merged**